jenetics/src/main/java/io/jenetics/UniformOderBasedCrossover.java

```diff
@@ -0,0 +1,64 @@
+package at.jku.dke.harmonic.optimizer.optimization.jenetics.jeneticsMO.jeneticsExtensions;
+
+import io.jenetics.Crossover;
+import io.jenetics.EnumGene;
+import io.jenetics.util.MSeq;
+
+import java.util.*;
+
+public class UniformOderBasedCrossover<T, C extends Comparable<? super C>> extends Crossover<EnumGene<T>, C> {
+
+    public UniformOderBasedCrossover(double probability) {
+        super(probability);
+    }
+
+    @Override
+    protected int crossover(MSeq<EnumGene<T>> that, MSeq<EnumGene<T>> other) {
+        if (that.length() != other.length()) {
+            throw new IllegalArgumentException(String.format("Required chromosomes with same length: %s != %s", that.length(), other.length()));
```

Comment:
Try to make the lines not to long. This makes reading the code unnecessary hard. Make lines not much longer then 80 chars.